playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -69,10 +69,11 @@
     - skip_ansible_lint
     - update_airship_osh_site
 
-- name: Copy site to Shipyard
+- name: Link shipyard to expected location
```

Comment:
CI is failing with "refusing to convert from directory to symlink for /opt/airship/shipyard". I am not familiar with the situation or the code's intention but maybe a "when: "{{ treasuremap_dir }}/{{ pegleg_output }} != {{ upstream_repos_clone_folder }}/airship/shipyard"" condition is a sufficient fix.

Reply:
I am not too familiar either. I will convert this to a recursive copy instead, until this is clarified.

Reply:
Its better to copy this file in shipyard folder as this file is made available to shipyard CLI container via mount. And we need read permissions for others to be accessible in the container. See PR #627 . Not sure how symlink is going to play out in all this.